DEVELOPMENT.md

```diff
@@ -249,6 +213,24 @@ check the
 page to ensure that all services are up and running (and not blocked by a quota
 issue, for example).
 
+### Deploy Knative Ingress
+
+Knative supports Istio, Contour, and Kourier as the Ingress solution.
```

Comment:
I'd say we support a variety of ingress solutions, this list makes it seem like these are the only choices, which is unfair to the folks working on Gloo, Kong, and Ambassador

Reply:
Done